movie_rec/lib/MovieReviews.py

```diff
@@ -1,23 +1,55 @@
 import json
+import StringIO
 from collections import defaultdict
+import psycopg2
 
 class MovieReviews(object):
-    def __init__(self, input_path, positive, negative, exclude=None):
-        self.reviews = [json.loads(line) for line in self.load_data(input_path)]
-        self.liked_movies = self.load_data(positive)
-        self.disliked_movies = self.load_data(negative)
+    def __init__(self, target_user=None):
+        self.reviews = self.load_reviews()
+        self.liked_movies = self.load_movies('1', target_user)
+        self.disliked_movies = self.load_movies('-1', target_user)
         self.seen_movies = self.liked_movies + self.disliked_movies
         self.critics = list(set([review['critic'] for review in self.reviews]))
         self.critic_ratings = self.critic_rating_mapping()
 
-        if exclude:
-            self.critic_ratings.pop(exclude, None)
-            self.critics.remove(exclude)
+        if target_user:
+            self.critic_ratings.pop(target_user, None)
+            self.critics.remove(target_user)
 
         self.movie_mapping = self.make_movie_mapping()
         self.all_movies = self.movie_mapping.keys()
-        self.create_predict_svm()
-        self.create_svm()
+        self.predict_svm = self.create_predict_svm()
+        self.training_svm = self.create_svm()
+
+
+    def load_reviews(self):
+        conn = psycopg2.connect('dbname=movie_rec') 
+        cur = conn.cursor()  
+        query = """select movie_id, user_id, rating from ratings"""
+        cur.execute(query)
+        results = cur.fetchall()
+        cur.close()
+        conn.close()
+        return [{
+            'movie': result[0],
+            'critic': result[1],
+            'rating': result[2]
+        } for result in results]
+
+
+    def load_movies(self, classification, target_user):
+        conn = psycopg2.connect('dbname=movie_rec') 
+        cur = conn.cursor()  
+        query = """select movie_id from ratings
+                 where user_id = %s and 
```

Comment:
What happens here when `target_user` is `None`? Would that not be equivalent to `...where user_id = NULL`?